app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
+  belongs_to :post, foreign_key: :post_id
+
+  def update_comments_count
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
[OPTIONAL] 

Here, we can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_count` method every time a new post has been saved to the database, then make the `update_comments_count` method private. Your model would end up looking like this:

```ruby
# app/models/comment.rb
  after_save :update_comment_counter

  private

  def update_comment_counter
    post.increment!(:comments_counter)
  end
  ```